routers/web/web.go

```diff
@@ -959,6 +959,7 @@ func RegisterRoutes(m *web.Route) {
 			m.Get("/commit/{sha:[a-f0-9]{7,40}}.{ext:patch|diff}", repo.RawDiff)
 		}, repo.MustEnableWiki, func(ctx *context.Context) {
 			ctx.Data["PageIsWiki"] = true
```

Comment:
```suggestion
```

Reply:
I think that's the last and correct one, we should keep it